src/main/resources/db/migration/V1__create_client_table.sql

```diff
@@ -1,11 +1,11 @@
 CREATE TABLE `client` (
-    clientId int AUTO_INCREMENT NOT NULL,
+    clientId int AUTO_INCREMENT NOT NULL PRIMARY KEY,
     name nvarchar(50) NOT NULL,
-    addressLine1 nvarchar(50) NOT NULL,
-    addressLine2 nvarchar(50),
-    town nvarchar(50),
-    city nvarchar(50),
-    postcode nvarchar(10),
-    country char(2) NOT NULL
```

Comment:
why is this only 2 chars

Reply:
ISO3166 country code as specified in pr JTD-001